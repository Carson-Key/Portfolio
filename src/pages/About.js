// Components
import Container from '../components/Container'
// Assests
import CarsonPicture from '../assets/Profile-Picture.png'
import BendiPicture from '../assets/Profile-Kitty.png'

const About = () => {
    return (
		<Container>
			<img className="float-left w-1/2 my-2 mx-2" src={CarsonPicture}/>
			<h3 className="font-bold">About Me</h3>
			<p>I'm a software engineer currently based out of Bend Oregon, also born and raised a Bendite! I went to Oregon State University Casacdes Campus, where I got a Bachelors of Science in Computer Science with an emphasis on Software Engineering. However, I've been programming for well over 10 years, and was fortunate enought to find my passion for programming at a fairly young age. In 2018 I took an intern position at CBT Nuggets, where I learned React and instantly fell in love with the framework. Ever since I've tried to use React in every area of my life I can, and hope to find a career as a React <img className="float-right w-1/2 my-2 mx-2" src={BendiPicture}/> programmer. Although my skills and education are in all aspects of programming. I love to learn new and exsiting technologies, with my current sights set on learning Next.js to apply my past in PHP with my love for React. I crrently live with my adorable kitten Bendi who, as you can see in her portrait, is quite the rascal.</p>
		</Container>
	)
}

export default About
