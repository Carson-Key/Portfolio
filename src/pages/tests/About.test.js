// Packages
import renderer from 'react-test-renderer'
// Pages
import About from '../About'

describe('About Page', () => {
    it('Should Render About page', () => {
        const about = renderer.create(
            <About />
        ).toJSON()
        expect(about).toMatchSnapshot()
    })
})