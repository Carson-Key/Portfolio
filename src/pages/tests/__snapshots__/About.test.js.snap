// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Page Should Render About page 1`] = `
<main>
  <p>
    About
  </p>
</main>
`;
